NameGame/NameGame/APIService.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  APIService.swift
+//  NameGame
+//
+//  Created by Tammy Le on 6/12/17.
+//  Copyright © 2017 WillowTree Apps. All rights reserved.
+//
+
+import Foundation
+
+/**
+    Reads and models data from online API.
+ */
+
+class APIService {
```

Comment:
So, a few things we should do here. First lets make are API service class name more descriptive. How about `UserRecordService`? Second thing, since we have no intention to subclass this service let's mark it `final`. By default you should mark all class you create as final unless you have an immediate need to subclass them.